.mint/python-binary-builder.yml

```diff
@@ -11,6 +9,31 @@ concurrency-pools:
     on-overflow: queue
 
 tasks:
+  - key: lookup-versions
+    docker: true
+    run: |
+      container="python:${{ init.python-version }}-alpine3.20"
+      docker pull $container
+      docker run $container pip --version | tee pip-output.txt
+      docker run $container pip show setuptools | tee setuptools-output.txt
+      grep -E '^pip [0-9\.]+ from' pip-output.txt | awk '{print $2}' | tee $MINT_VALUES/pip-version
+      grep -E '^Version: [0-9\.]+' setuptools-output.txt | awk '{print $2}' | tee $MINT_VALUES/setuptools-version
+      pip_version=$(cat $MINT_VALUES/pip-version)
+      setuptools_version=$(cat $MINT_VALUES/setuptools-version)
+      echo "${{init.python-version}},$pip_version,$setuptools_version" | tee python-3.12.5.txt
```

Comment:
Should this be `tee python-${{ init.python-version }}.txt`?

Reply:
Looking from my phone, but I think yes, but this file is actually unused now?